src/layer/x86/convolutiondepthwise_3x3_int8.h

```diff
@@ -411,4 +411,5 @@ static void convdw3x3s2_int8_requant_sse(const Mat &bottom_blob, Mat &top_blob,
             r2 += tailstep;
         }
     }
-}
\ No newline at end of file
+}
+
```

Comment:
keep the unix line endings

Reply:
OK, mac vim editor problem. It has been solved.